### PR TITLE
Feat/civ 6406

### PR DESCRIPTION
**Pull request by maciejmodzelewski**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-6406

### Change description ###
Reference to entering the PBA details removed in nonprod and PageIDs replaced.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/steps_file_LRspec.js [hunk 1]
```diff
@@ -332,8 +330,6 @@ module.exports = function () {
                  () => specInterestDateStartPage.selectInterestDateStart(),
                  () => specInterestDateEndPage.selectInterestDateEnd(),
                  () => this.clickContinue(),
-                 () => pbaNumberPage.selectPbaNumber(),
```
Comment (m-meulendijks-v1) [inline e2e/steps_file_LRspec.js:335]: I wonder if we should put the changes in tests under a featureToggle.
There is a variable you could use to check if pbav3 is on: const {checkToggleEnabled} = require('./testingSupport');

File: e2e/steps_file_LRspec.js [hunk 2]
```diff
@@ -15,8 +15,6 @@ const addAnotherDefendant = require('./pages/createClaim/addAnotherDefendant.pag
 const respondent2SameLegalRepresentative = require('./pages/createClaim/respondent2SameLegalRepresentative.page');
```
Comment (m-meulendijks-v1) [inline e2e/steps_file_LRspec.js:15]: This file is for specified end to end tests only. There should be another one for unspecified tests called steps_file.js.

In the api folder, there are also a couple steps files for the api tests, you might have to make some changes there.

File: e2e/steps_file_LRspec.js (hunk 1 above)
Comment (maciejmodzelewski) [reply e2e/steps_file_LRspec.js:335]: Done

File: e2e/steps_file_LRspec.js (hunk 2 above)
Comment (maciejmodzelewski) [reply e2e/steps_file_LRspec.js]: Done

**Review by drummondjm — APPROVED** [review]

**Review by m-meulendijks-v1 — APPROVED** [review]

**Pull request closed without merging**